std_msgs/Header header
geometry_msgs/Point[] pose_keypoints
geometry_msgs/Point[] hands_keypoints
std_msgs/String[] handednesses